<ui version="4.0" >
 <class>PSFitWinBase</class>
 <widget class="QDialog" name="PSFitWinBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Fit settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QComboBox" name="histoType" >
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="loopSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>24</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="loopLabel" >
         <property name="text" >
          <string>on loop</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="loopSel" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="FitType" >
       <property name="currentIndex" >
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="groupBox3" >
       <property name="title" >
        <string>Chi^2 settings</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="ChiLabel" >
              <property name="text" >
               <string>Chi^2 cut-off</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Spacer2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="ChiCut" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum" >
               <number>999999999</number>
              </property>
              <property name="value" >
               <number>100000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="divndfBox" >
            <property name="text" >
             <string>store chi^2/ndf (off: store raw chi^2)</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="refitBox" >
            <property name="text" >
             <string>fit only pixels with chi2>cut from previous fit</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="LimitGroup" >
       <property name="title" >
        <string>Limited fit range (min=max=0 -> no limit)</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="TextLabel1_2_2" >
            <property name="text" >
             <string>Minimum</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="FitXmin" >
            <property name="minimum" >
             <double>-99999999.000000000000000</double>
            </property>
            <property name="maximum" >
             <double>99999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel1_2_3" >
            <property name="text" >
             <string>Maximum</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="FitXmax" >
            <property name="minimum" >
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum" >
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="ParameterBox" >
       <property name="title" >
        <string>Initial parameter settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QCheckBox" name="guessParBox" >
            <property name="text" >
             <string>guess initial parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="parLabel1" >
                <property name="text" >
                 <string>Par. 1</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacerPar1" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>123</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="parBox1" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer2Par1" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="fixPar1" >
                <property name="text" >
                 <string>fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="parLabel2" >
                <property name="text" >
                 <string>Par. 2</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacerPar2" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>123</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="parBox2" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer2Par2" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="fixPar2" >
                <property name="text" >
                 <string>fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="parLabel3" >
                <property name="text" >
                 <string>Par. 3</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacerPar3" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>123</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="parBox3" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer2Par3" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="fixPar3" >
                <property name="text" >
                 <string>fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="parLabel4" >
                <property name="text" >
                 <string>Par. 4</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacerPar4" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>130</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="parBox4" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer2Par4" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="fixPar4" >
                <property name="text" >
                 <string>fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="parLabel5" >
                <property name="text" >
                 <string>Par. 5</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacerPar5" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>130</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="parBox5" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer2Par5" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="fixPar5" >
                <property name="text" >
                 <string>fix</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="vcalBox" >
            <property name="text" >
             <string>convert VCAL to charge automatically</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="ErrorBox" >
       <property name="title" >
        <string>Errors</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="binomialBox" >
            <property name="text" >
             <string>Use binomial errors (for occ. data)</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="fromHistoBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Get errors from another histogram:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer name="spacer6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>31</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="errorCBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="fractBox" >
              <property name="text" >
               <string>Use fraction of value as err.:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer9" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>41</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="fractEdit" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>0.05</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="OKButt" >
         <property name="text" >
          <string>Start Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancButt" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>fromHistoBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>errorCBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>40</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel" >
     <x>78</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fractBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fractEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>41</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel" >
     <x>275</x>
     <y>632</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixPar1</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel" >
     <x>402</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixPar2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>48</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel" >
     <x>405</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixPar3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel" >
     <x>428</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixPar4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>78</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel" >
     <x>397</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixPar5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>106</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel" >
     <x>403</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>parBox1</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>38</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel" >
     <x>290</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>parBox2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>59</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel" >
     <x>293</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>parBox3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>24</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>parBox4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>142</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>279</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guessParBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>parBox5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>167</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
